build/IPDK_Container/README_DOCKER.md

```diff
@@ -20,11 +19,11 @@ the example P4 pipeline programs.
 
 ## Install IPDK CLI
 To fully use all the features of IPDK container and if you want to follow the
-examples, then install the IPDK CLI with the following command executed from 
+examples, then install the IPDK CLI with the following command executed from
 the directory this README is found in:
 
 ```
-./ipdk install
+./scripts/ipdk.sh install
```

Comment:
I liked the cleanliness of the `./ipdk` next to `ipdk` as command if the install to `~/.local/bin` or `~/bin` couldn't be done but I can understand your hesitation. No problem!